timeless/message/property_resource.py

```diff
@@ -15,5 +21,18 @@ class PropertyResource(MessageResource):
 
     """
 
+    directory = None
+    locale = None
+
+    def __init__(self, **kwargs):
+        self.directory = kwargs.get("directory")
+        self.locale = kwargs.get("locale")
+
     def get(self, key):
-        raise Exception("PropertyResources not implement yet!")
+        path = os.path.join(app.root_path, self.directory)
+        config = ConfigParser()
+        config.read(f"{path}message_{self.locale}.properties")
+        if not config.get("MESSAGES", key):
+            raise Exception("Value not found for key")
```

Comment:
@paulodamaso Should it throw an exception, do we make it that strong? Wouldn't it be better to just return null or empty string?

Reply:
@amihaiemil Well, null is out of question, and empty String is not really adequate too because we can have declared properties with a empty string as value